### ISSUE TITLE
Downloading Images

### ISSUE THREAD

**Issue by SunburnedGoose** [kyleshay/DIMDefs#1]
https://github.com/kyleshay/DIM/blob/angular-port/build/processBungieManifest.js

Dunno if you saw this but I am using node to parse the manifest and download the files before building the app.  This way they are cached locally and images won't break.

**Comment by kyleshay** [comment]
oh awesome -- didn't see that you had a script to process the manifest file already

**Comment by SunburnedGoose** [comment]
I'm fine with having a dedicated repo if you want to do this.  I'll merge my changes.

**Comment by kyleshay** [comment]
yeah, i mean -- instead of merging you could create a branch for the angular version, since i'm using this version for v2. whichever way is fine, we can have it livei n the project as is if we wanted as well. no preference.
